CONTRIBUTING.md

```diff
@@ -14,6 +14,8 @@ conda env create -f environment.yml
 
 # install PyTorch with specific CUDA version
 conda install pytorch torchvision cudatoolkit=<YOUR_CUDA_VERSION> -c pytorch
```

Comment:
It seems that we have include pytorch and tensorflow in the `environment.yml`. You can remove the `conda install ...` line,

Reply:
> It seems that we have include pytorch and tensorflow in the `environment.yml`. You can remove the `conda install ...` line,

Sure, BTW, tensorflow-serving-api is also included in `environment.yml`, I'll remove this line too